app/models/graph/schema.rb

```diff
@@ -23,8 +25,5 @@ module Graph
       model = Object.const_get(gid.model_name)
       Graph::FindLoader.for(model).load(gid.model_id.to_i)
     end
-
-    lazy_resolve(Promise, :sync)
-    instrument(:query, GraphQL::Batch::Setup)
```

Comment:
Your gem includes the right setup? What happens if `instrument(:query, GraphQL::Batch::Setup)` is included twice by accident? 🤔